wandb/sdk/launch/_project_spec.py

```diff
@@ -78,7 +78,9 @@ def __init__(
         # the builder key can be passed in through the resource args
         # but these resource_args are then passed to the appropriate
         # runner, so we need to pop the builder key out
-        resource_args_build = resource_args.get(resource, {}).pop("builder", {})
+        resource_args_build = {}
+        if resource_args.get(resource):
```

Comment:
This feels weird to me, and is hiding another more nefarious bug somewhere that causes the resource to be an empty string. (Maybe its just old queues without a DRC). At any rate, I don't think we should do this. I think if this is something coming through as a string we should find that issue and solve it.

Reply:
Yup, super valid.